infoschema/metric_table_def.go

```diff
@@ -1835,4 +1835,208 @@ var MetricTableMap = map[string]MetricTableDef{
+		Comment: "node disk write latency(ms)",
+	},
+	"node_disk_read_latency": {
+		PromQL:  `(rate(node_disk_read_time_seconds_total{$LABEL_CONDITIONS}[$RANGE_DURATION])/ rate(node_disk_reads_completed_total{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels:  []string{"device", "instance"},
+		Comment: "node disk read latency(ms)",
+	},
+	"node_disk_throughput": {
+		PromQL:  `irate(node_disk_read_bytes_total{$LABEL_CONDITIONS}[$RANGE_DURATION]) + irate(node_disk_written_bytes_total{$LABEL_CONDITIONS}[$RANGE_DURATION])`,
+		Labels:  []string{"device", "instance"},
+		Comment: "Units is byte",
+	},
+	"node_filesystem_space_used": {
+		PromQL:  `((node_filesystem_size_bytes{$LABEL_CONDITIONS} - node_filesystem_avail_bytes{$LABEL_CONDITIONS}) / node_filesystem_size_bytes{$LABEL_CONDITIONS}) * 100`,
+		Labels:  []string{"device", "instance"},
+		Comment: "Filesystem used space. If is > 80% then is Critical.",
+	},
+	"node_file_descriptor_allocated": {
+		PromQL: `node_filefd_allocated{$LABEL_CONDITIONS}`,
+		Labels: []string{"instance"},
+	},
+	"node_network_in_drops": {
+		PromQL: `rate(node_network_receive_drop_total{$LABEL_CONDITIONS}[$RANGE_DURATION]) `,
+		Labels: []string{"device", "instance"},
+	},
+	"node_network_out_drops": {
+		PromQL: `rate(node_network_transmit_drop_total{$LABEL_CONDITIONS}[$RANGE_DURATION])`,
+		Labels: []string{"device", "instance"},
+	},
+	"node_network_in_errors": {
+		PromQL: `rate(node_network_receive_errs_total{$LABEL_CONDITIONS}[$RANGE_DURATION])`,
+		Labels: []string{"device", "instance"},
+	},
+	"node_network_out_errors": {
+		PromQL: `rate(node_network_transmit_errs_total{$LABEL_CONDITIONS}[$RANGE_DURATION])`,
+		Labels: []string{"device", "instance"},
+	},
+	"node_network_in_traffic": {
+		PromQL: `rate(node_network_receive_bytes_total{$LABEL_CONDITIONS}[$RANGE_DURATION]) or irate(node_network_receive_bytes_total{$LABEL_CONDITIONS}[$RANGE_DURATION])`,
+		Labels: []string{"device", "instance"},
+	},
+	"node_network_out_traffic": {
+		PromQL: `rate(node_network_transmit_bytes_total{$LABEL_CONDITIONS}[$RANGE_DURATION]) or irate(node_network_transmit_bytes_total{$LABEL_CONDITIONS}[$RANGE_DURATION])`,
+		Labels: []string{"device", "instance"},
+	},
+	"node_network_in_packets": {
+		PromQL: `rate(node_network_receive_packets_total{$LABEL_CONDITIONS}[$RANGE_DURATION]) or irate(node_network_receive_packets_total{$LABEL_CONDITIONS}[$RANGE_DURATION])`,
+		Labels: []string{"device", "instance"},
+	},
+	"node_network_out_packets": {
+		PromQL: `rate(node_network_transmit_packets_total{$LABEL_CONDITIONS}[$RANGE_DURATION]) or irate(node_network_transmit_packets_total{$LABEL_CONDITIONS}[$RANGE_DURATION])`,
+		Labels: []string{"device", "instance"},
+	},
+	"node_network_interface_speed": {
+		PromQL:  `node_network_transmit_queue_length{$LABEL_CONDITIONS}`,
+		Labels:  []string{"device", "instance"},
+		Comment: "node_network_transmit_queue_length = transmit_queue_length value of /sys/class/net/<iface>.",
```

Comment:
Are `node_network_interface_speed` and `node_network_transmit_queue_length` same thing?

Reply:
The comment was from `tidb-ansible`.
https://github.com/pingcap/tidb-ansible/blob/master/scripts/node.json#L7477